interface/forms/annotate_diagram/mapdiagram/js/sketch.js

```diff
@@ -0,0 +1,182 @@
+var __slice = Array.prototype.slice;
```

Comment:
Yes

Sent from my iPhone

> On Sep 1, 2016, at 12:03 AM, Brady Miller <notifications@github.com> wrote:
> 
> In interface/forms/annotate_diagram/mapdiagram/js/sketch.js:
> 
> > @@ -0,0 +1,182 @@
> > +var __slice = Array.prototype.slice;
> Is this a third party library?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>